pages/analysis.php

```
@@ -43,7 +43,7 @@
 if (
     !is_array($phpstanResult)
     || !is_array($phpstanResult['files'])
-    || (array_key_exists('errors', $phpstanResult) && count($phpstanResult['errors']) > 0)
+  //  || (array_key_exists('errors', $phpstanResult) && count($phpstanResult['errors']) > 0)
```

ignored errors werden auch als error returned.. müsste man abfiltern